app/equipment/weapon.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from app.equipment.equipment import Equipment
+
+
+class Weapon(Equipment):
+    def __init__(self, name: str, power: int) -> None:
+        super().__init__(name)
+        self.power = power
+
+    @classmethod
+    def apply(cls, knight: Any, source: dict) -> None:
+        weapon_data = (
+            source.get(
+                knight.name.lower(),
+                source.get(
+                    knight.name.replace(" ", "_").lower(), {}
+                )
+            ).get("weapon", None)
+        )

```

Comment:
What do you do here?
You loop through all knights and their configs here
![image](https://github.com/user-attachments/assets/31c6c5a0-74a1-414f-9be1-bb8a4f25dd95)

This is weapon dict of specific knight. So you just need to fetch power and add it to knight's power, ain't I right?
```
"weapon": {
        "name": "Two-handed Sword",
        "power": 55,
},
```